website/docusaurus.config.js

```diff
@@ -65,6 +65,11 @@ module.exports = {
                 label: 'API',
                 position: 'left',
                 docId: 'api',
+            }, {
+                type: 'doc',
+                label: 'Flowchart',
+                position: 'left',
+                docId: 'flowchartsoverview',
```

Comment:
I don't think we want to have this part of the main menu. Let's drop a section into the [contributing guidelines](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) and link to the page.

Reply:
@christian-bromann, Should we position the flowchart above the "Proposing a Change" section, or do you think it should be relocated under a different section? Please share your thoughts.

![Screenshot 2023-09-07 at 9 42 06 PM](https://github.com/webdriverio/webdriverio/assets/49207189/e9c0413f-842a-4bea-89b9-e6770a812260)

Reply:
This looks good to me 👍